tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = flake8, pylint, check-{py27,py36}
+envlist = flake8, pylint, check-{py27,py3}
 skip_missing_interpreters = True
```

Comment:
This needs to be overridden when running automatically to ensure that all python interpreters are tested